### ISSUE TITLE
Allow an external participant to give or request control documentation incorrect

### ISSUE THREAD

**Issue by tkrancz** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4324]
Under "Allow an external participant to give or request control" you write this: "This setting controls whether external participants in a meeting can give control of their shared desktop or window to other participants in the meeting." 

I believe this is incorrect. Based on my tests this rather controls, if internal user can give control to external users. If "Allow an external participant to give or request control" is set to FALSE, then internal user cannot give control to external users, and external users cannot request control from internal users. Regardless of how "Allow an external participant to give or request control" is set, external users can give control to their shared application/screen to other external users or to internal users. So, if this setting is set to FALSE, it prevents external users from getting control over internal users' screen/application.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dc665475-f843-e4c1-0ca6-0b83873abb54
* Version Independent ID: 95d32598-c920-ed5c-d665-69394528ca27
* Content: [Manage meeting policies - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-in-teams#feedback)
* Content Source: [Teams/meeting-policies-in-teams.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/meeting-policies-in-teams.md)
* Service: **msteams**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by scanum** [comment]
@tkrancz Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by tkrancz** [comment]
Thank you. Also there is a problem with this sentence: "To use PowerShell to control whether external participants can give control or accept requests for control, use the AllowExternalParticipantGiveRequestControl cmdlet."

AllowExternalParticipantGiveRequestControl is not a cmdlet, but it is only a parameter of a cmdlet

**Comment by Reeced40** [comment]
@tonysmit please could you assist.

**Comment by tonysmit** [comment]
@LanaChin - can you make these changes please to the doc? Thanks, Tony

**Comment by marianocalero** [comment]
@LanaChin Hello Lana! Just to confirm, I noticed this was closed 13 days ago, but the Docs are still not reflecting what the parameter controls.

"Allow an external participant to give or request control – lets guests and external (federated) users give or request control of the presenter's desktop or application."

-https://docs.microsoft.com/en-us/microsoftteams/configure-desktop-sharing

"This is a per-user policy. This setting controls whether external participants in a meeting can give control of their shared desktop or window to other participants in the meeting."

-https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-in-teams#allow-an-external-participant-to-give-or-request-control

The above two descriptions are from two different Docs pages describing the parameter in the title. It is a little confusing as it does not seem to accurately describe what the parameter controls.

This should read something like: "This is a per-user policy. Whether an organization has this set for a user does not control what external participants can do, regardless of what the Meeting Organizer has set. This parameter controls whether external participants can be given control or request control of the sharer's screen, depending on what the sharer has set within their organization's meeting policies."

The way that it is currently written makes it seem as though the parameter will allow the meeting organizer from an organization to control whether an external participant that is sharing their screen can give control to an internal participant, but this is controlled by the external participant's own meeting policies from their organization.
